IPMOF_emap.py

```diff
@@ -28,24 +28,23 @@
     # Create MOF list by reading mol2 files from a directory
     mof_list = get_mof_file_list(sim_dir['mol2_dir'], 'mol2', force_field)
 
+# Calculate atom list for remaining MOFs
+atom_list = get_uniq_atom_list(mof_list[base_mof_index:])
+
```

Comment:
don't see how this could work since base_mof_index is defined below...

fixed in a later commit?